### PR TITLE
Docs: Improve documentation website

### PR DESCRIPTION
**Pull request by ikun-Lg**
1.Improve documentation website
Snipaste_2024-03-20_00-27-53

### CODE REVIEW

File: docs/imgs/Wrappingup.png
Comment (jianyi-gronk) [inline docs/imgs/Wrappingup.png:1]: redundant irrelevant pictures

**Review by jianyi-gronk — COMMENTED** [review]
It is recommended to delete irrelevant pictures

File: docs/guide/dubboForWEB/Server-SideRendering.md
Comment (jianyi-gronk) [inline docs/guide/dubboForWEB/Server-SideRendering.md:1]: Might be better named ServerSideRendering.md

Comment (ikun-Lg) [reply docs/guide/dubboForWEB/Server-SideRendering.md:1]: get

File: docs/imgs/Wrappingup.png
Comment (ikun-Lg) [reply docs/imgs/Wrappingup.png:1]: get

**Review by jianyi-gronk — APPROVED** [review]
LGTM.

**Pull request merged**